### PR TITLE
LPS-133405 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133405

### CODE REVIEW

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you move the case to WebContentDisplayWithXSS.testcase?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesWithXSS.testcase [hunk 1]
```diff
@@ -24,6 +24,79 @@ definition {
...
+		}
+
+		task ("View the XSS is escaped") {
+			var actualScript = '''<script>alert(123);</script>''';
+
+			var escapedScript = '''&lt;script&gt;alert(123);&lt;/script&gt;''';
+
+			LexiconCard.viewCardNotPresent(card = "${actualScript}");
+
+			LexiconCard.viewCardTitle(card = "${escapedScript}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesWithXSS.testcase:54]: Use WebContentTemplate.viewCardsEntryCP

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro [hunk 2]
```diff
@@ -1819,9 +1819,16 @@ definition {
...
+		if (isSet(actualScript)) {
+			AssertTextEquals(
+				locator1 = "WCEditWebContent#SELECTED_TEMPLATE",
+				value1 = "${actualScript}");
+		}
+		else {
+			AssertTextEquals(
+				locator1 = "WCEditWebContent#SELECTED_TEMPLATE",
+				value1 = "${templateName}");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro:1831]: Could you use
```
		var templateName = StringUtil.replace("${templateName}", "$lt;", "<");
		var templateName = StringUtil.replace("${templateName}", "$gt;", ">");

		AssertTextEquals(
			locator1 = "WCEditWebContent#SELECTED_TEMPLATE",
			value1 = "${templateName}");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesWithXSS.testcase [hunk 3]
```diff
@@ -24,6 +24,79 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-117985. View the Web Content Template with XSS on widget page."
+	@priority = "4"
+	@uitest
+	test ViewWebContentTemplate {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesWithXSS.testcase:30]: SelectWebContentBasedOnTemplateNameWithXSS

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you move the case to WebContentDisplayWithXSS.testcase?

Hi @Tim-Cao Sure, PR Updated following your comments.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/2a9251609f7addfa651155510168e602b3c02138...aa629ac78b41077d148d4b6ae4b9b1d2d2a08021

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**